applications/crossbar/src/modules_v2/cb_phone_numbers_v2.erl

```diff
@@ -857,8 +857,7 @@ get_prefix(City) ->
             ReqParam = wh_util:uri_encode(binary:bin_to_list(City)),
             Req = binary:bin_to_list(<<Url/binary, "/", Country/binary, "/city?pattern=">>),
             Uri = lists:append(Req, ReqParam),
-            case ibrowse:send_req(Uri, [], 'get') of
-                {'error', _Reason}=E -> E;
+            case kz_http:req('get', Uri, []) of
```

Comment:
xref says you forgot to export `req/2`. Also, why use `kz_http:req('get', Uri, [])` instead of the shorter `kz_http:req('get', Uri)`?